conjurapi/router_v5.go

```diff
@@ -50,11 +50,47 @@ func (r RouterV5) CheckPermissionRequest(resourceID, privilege string) (*http.Re
 	)
 }
 
+func (r RouterV5) ResourceRequest(resourceID string) (*http.Request, error) {
+	account, kind, id, err := parseID(resourceID)
+	if err != nil {
+		return nil, err
+	}
+
+	requestURL := makeRouterURL(r.resourcesURL(account), kind, url.QueryEscape(id))
+
+	return http.NewRequest(
+		"GET",
+		requestURL.String(),
+		nil,
+	)
+}
+
+func (r RouterV5) ResourcesRequest(filter *ResourceFilter) (*http.Request, error) {
+
+	var query []string
```

Comment:
Can this be const

Reply:
I don't think so. The value of `query` depends on what's in `filter`. `filter` might be `nil` at runtime, so there's no way to set this at compile time.